src/mito/src/manifest/action.rs

```diff
@@ -12,184 +12,4 @@
-    fn test_encode_decode_action_list() {
-        common_telemetry::init_default_ut_logging();
-        let mut protocol = ProtocolAction::new();
-        protocol.min_reader_version = 1;
-
-        let table_info = RawTableInfo::from(test_util::build_test_table_info());
-
-        let mut action_list = TableMetaActionList::new(vec![
-            TableMetaAction::Protocol(protocol.clone()),
-            TableMetaAction::Change(Box::new(TableChange { table_info })),
-        ]);
-        action_list.set_prev_version(3);
-
-        let bs = action_list.encode().unwrap();
-
-        logging::debug!(
-            "Encoded action list: \r\n{}",
-            String::from_utf8(bs.clone()).unwrap()
-        );
-
-        let e = TableMetaActionList::decode(&bs, 0);
-        assert!(e.is_err());
-        assert_eq!(
-            "Manifest protocol forbid to read, min_version: 1, supported_version: 0",
-            format!("{}", e.err().unwrap())
-        );
-
-        let (decode_list, p) = TableMetaActionList::decode(&bs, 1).unwrap();
-        assert_eq!(decode_list, action_list);
-        assert_eq!(p.unwrap(), protocol);
-    }
-
-    // These tests are used to ensure backward compatibility of manifest files.
-    // DO NOT modify the serialized string when they fail, check if your
-    // modification to manifest-related structs is compatible with older manifests.
-    #[test]
-    fn test_table_manifest_compatibility() {
-        let table_change = r#"{"table_info":{"ident":{"table_id":0,"version":0},"name":"demo","desc":null,"catalog_name":"greptime","schema_name":"public","meta":{"schema":{"column_schemas":[{"name":"host","data_type":{"String":null},"is_nullable":false,"is_time_index":false,"default_constraint":null,"metadata":{}},{"name":"cpu","data_type":{"Float64":{}},"is_nullable":true,"is_time_index":false,"default_constraint":null,"metadata":{}},{"name":"memory","data_type":{"Float64":{}},"is_nullable":false,"is_time_index":false,"default_constraint":null,"metadata":{}},{"name":"ts","data_type":{"Timestamp":{"Millisecond":null}},"is_nullable":true,"is_time_index":true,"default_constraint":null,"metadata":{"greptime:time_index":"true"}}],"timestamp_index":3,"version":0},"primary_key_indices":[0],"value_indices":[1,2,3],"engine":"mito","next_column_id":1,"region_numbers":[],"engine_options":{},"options":{"write_buffer_size":null,"ttl":null,"extra_options":{}},"created_on":"2023-03-06T08:50:34.662020Z"},"table_type":"Base"}}"#;
-        serde_json::from_str::<TableChange>(table_change).unwrap();
-
-        let table_remove =
-            r#"{"table_ident":{"table_id":42,"version":0},"table_name":"test_table"}"#;
-        serde_json::from_str::<TableRemove>(table_remove).unwrap();
-
-        let protocol_action = r#"{"min_reader_version":0,"min_writer_version":1}"#;
-        serde_json::from_str::<ProtocolAction>(protocol_action).unwrap();
-    }
-}
+pub use storage::manifest::{TableChange, TableMetaAction, TableMetaActionList, TableRemove};
```

Comment:
you can use "git mv" to move this file to another folder. reduce CR size. then the "pub use" here is not applicable, maybe the call sites need to be changed